### PR TITLE
Initial Homepage Design

### PR DESCRIPTION
**Pull request by kerrylewis**
Created a first draft for the homepage - any feedback is appreciated! Mostly changed styling of background image and navigation bar, as well as including a brief summary of Inspicio.  The Inspicio logo and favicon have also been added. Any feedback is appreciated!

### CODE REVIEW

File: Inspicio/Inspicio.csproj [hunk 1]
```diff
@@ -20,12 +20,26 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Content Include="wwwroot\css\home.css" />
     <Content Include="wwwroot\css\JavaScript.js" />
     <Content Include="wwwroot\css\thumbs.js" />
+    <Content Include="wwwroot\images\add.svg" />
+    <Content Include="wwwroot\images\create.svg" />
+    <Content Include="wwwroot\images\eye.jpg" />
+    <Content Include="wwwroot\images\Inspicio bar.png" />
```
Comment (wilka) [inline Inspicio/Inspicio.csproj]: Spaces in names of files served by the web server can be problematic, typically people use `-` instead. e.g. in this url https://stackoverflow.com/questions/497908/is-a-url-allowed-to-contain-a-space.

You can also run into problems with some web servers being case sensitive, so it's also common to use lowercase letters for everything.

File: Inspicio/Inspicio.csproj [hunk 2]
```diff
@@ -58,5 +72,6 @@
     <Folder Include="Migrations\" />
     <Folder Include="Migrations\" />
     <Folder Include="Migrations\" />
+    <Folder Include="Migrations\" />
```
Comment (wilka) [inline Inspicio/Inspicio.csproj]: Not sure what's going on here, but the Migrations folders is mentioned multiple times in the project file - I'd expect it to only be there once.

File: Inspicio/wwwroot/css/site.css [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
+    padding-bottom: 0px;
+    font-family:'Century Gothic';
+    padding-left: 0px; 
 }
 
 hr {
     border-color: #e6e6e6;
 }
+nav{
+    border-radius:0!important;
```
Comment (wilka) [inline Inspicio/wwwroot/css/site.css:12]: Is the `!important;` needed here?

**Review by wilka — CHANGES_REQUESTED** [review]
👍 on the look. Also, that Inspicio/wwwroot/images/Newcastle.jpg image follows me everywhere :)

File: Inspicio/Inspicio.csproj (hunk 1 above)
Comment (kerrylewis) [reply Inspicio/Inspicio.csproj]: Ah that file shouldn't even be in there! Thanks! I'll get deleting the random pictures (including Newcastle.jpg one haha!)

File: Inspicio/wwwroot/css/site.css (hunk 3 above)
Comment (kerrylewis) [reply Inspicio/wwwroot/css/site.css:12]: The "!important" is currently preventing a white box appearing beneath the nav bar. Ideally the nav bar will be fixed in the future, so might be better to change it later on once the other pages are sorted.

File: Inspicio/Inspicio.csproj (hunk 2 above)
Comment (kerrylewis) [reply Inspicio/Inspicio.csproj]: No idea... I'll get deleting again! Thanks for the spot!

**Review by wilka — APPROVED** [review]

**Pull request merged**